### ISSUE TITLE
Overlay not working on some custom grid's

### ISSUE THREAD

**Issue by tr0janman697** [boppreh/steamgrid#19]
First, thank you SO MUCH for your work on steamgrid.  It truly is one of the most helpful .exe's I have!

Now to the issue.  Some games like Kelly Slater Pro Surfer or Ken Griffey Jr Slugfest do not have proper 480x215 grids. The custom overlay via steamgrid folder works as I could see it scale with the image in minuscule in size.  When I attempt to replace the Kelly Slater Pro Surfer grid with my own custom .jpg, steamgrid will replace it back with the old pulled image from the website.  Steamgrid will not replace most custom .png replacement but now it does not install the custom overlay :( 

The notification for "Steamgrid has replaced 0 images and applied (all) amount of overlays"  will pop up, and it could of replaced 4-10 images and taken away the custom overlay.

A similar issue happened to me with SUPERHOT.  Steamgrid finds the standard one and apply's the overlay.  Yet when I add a custom grid, steamgrid won't replace the custom grid but dies not apply the overlay either. 

So to recap, steamgrid replaces certain games with custom images applied, and if it does not replace the custom image grid's it will certainly not apply an overlay on those specific games.

I don't really know where my log's are located but I am willing to help in anyway I can to fix this issue,  I love having overlays and it is quite strange that only some of these don't want to work.

THANK YOU SO MUCH AGAIN FOR ALL YOUR HARD WORK!

**Comment by boppreh** [comment]
I believe you accidentally created two issues. I'm closing this one.
